spec/sections/standard.adoc

```diff
@@ -554,18 +563,18 @@ seed.outputs.json:
 a|
 <1> Optional array containing elements defined by Files Elements sub-section.
 <2> Required string containing output identifier.
-<3> Required string containing IANA Media type of file.
-<4> Required string containing glob expression for file capture. Processing system is expected to
+<3> Optional string containing IANA Media type of file.
+<4> Required string containing glob expression for file capture. Executor† is expected to
 capture output relative to OUTPUT_DIR.
-<5> Optional string containing either a numeric count or `*` for unbounded output. Default value
-is `1`.
-<6> Optional boolean indicating whether processing system should assume failure if output data is missing. Default value
+<5> Optional boolean indicating whether a single or multiple values are supported. The executor† is expected to
+validate that matched outputs are consistent with this value - one or at least one. Default value is false.
```

Comment:
I would remove `The executor† is expected to validate that matched outputs are consistent with this value - one or at least one.` How an executor handles matching up inputs and outputs (like for recipes) is outside of the scope of Seed.